### ISSUE TITLE
ostrio logger console : TypeError: Cannot call method 'substring' of undefined

### ISSUE THREAD

**Issue by bletual** [VeliovGroup/Meteor-logger#1]
Hi,

since Meteor 1.3 update, there is an exception in debug.js, line 97 :+1: 

TypeError: Cannot call method 'substring' of undefined

var parentfile = p.split(appDir)[1].substring(1);

should be :+1: 

var parentfile = p.split(appDir)[0].substring(1);

Code is :+1: 

```
Debug.prototype.trace = function(msg, type) {
    var completeMessage = "",
        format = dateFormat(new Date(), 'HH:MM:ss'),
        func = undefined,
        appDir = path.dirname(require.main.filename),
        p = this.get_file_parent();

    if(type != "ERROR"){
        if (p) {
            var parentfile = p.split(appDir)[1].substring(1);
            func = " @ "+parentfile+":"+this.get_line_parent();
        } else {
            return;
        }
    }

    if(colors) typeF = eval(type.toLowerCase()+"(type)")
    if(!colors) typeF = type;

    if(type == "ERROR") func = "";

    // Filter process
    for(i=0;i<consoleFilter.length;i++){
        if(type==consoleFilter[i]) return;
    }

    if(type=="ERROR") completeMessage = completeMessage + "("+format+" - "+typeF+"  "+func+"            ) - "+msg;
    if(type=="LOG") completeMessage =   completeMessage + "("+format+" - "+typeF+" "+func+") - "+msg;
    if(type=="DEBUG") completeMessage = completeMessage + "("+format+" - "+typeF+" "+func+") - "+msg;
    if(type=="INFO") completeMessage =  completeMessage + "("+format+" - "+typeF+"  "+func+") - "+msg;
    if(type=="WARN") completeMessage =  completeMessage + "("+format+" - "+typeF+"  "+func+") - "+msg;
    logMessage = "("+format+") ["+type+""+func+"] - "+msg;


    // Special display for objects
    if(typeof(msg)=="object"){
        console.log("("+format+") ["+typeF+""+func+"] - Object: ");
        completeMessage = JSON.stringify(msg, null, 4);
    }
    console.log(completeMessage);
    if(logToFile) this.logFile(logMessage,type);
};
```

**Comment by dr-dimitru** [comment]
Related: [logger-console#1](https://github.com/VeliovGroup/Meteor-logger-console/issues/1)

**Comment by dr-dimitru** [comment]
Hi @bletual , thank you for contribution

This issue is fixed in [v1.1.0](https://github.com/VeliovGroup/Meteor-logger-console/releases/tag/1.1.0), please confirm on your end
